app/views/two_factor_authentication/webauthn_verification/show.html.erb

```diff
@@ -4,10 +4,8 @@
   <%= t('two_factor_authentication.webauthn_header_text') %>
 </h1>
 
-<div class='no-spinner'>
-  <div class='mt-tiny mb3'>
-    <%= @presenter.webauthn_help %>
-  </div>
+<div class='mt-tiny mb3'>
+  <%= @presenter.webauthn_help %>
 </div>
```

Comment:
I'm a bit confused by what's being shown here in the diff on GitHub, since the diff seems to imply there's an unbalanced `div` tag.. but in reality, it's not 🤔 

https://github.com/18F/identity-idp/blob/6a3fbc76383f3e735f228a8d0ed374cecf5f9182/app/views/two_factor_authentication/webauthn_verification/show.html.erb#L7-L11

Ultimately it doesn't make a difference since it's in good shape, but the diff seems to imply there had been a bug when there wasn't.